README.md

```diff
@@ -8,51 +8,22 @@
 [![LICENSE](https://img.shields.io/badge/license-Apache-blue.svg?style=flat)](https://www.apache.org/licenses/LICENSE-2.0)
 [![Contribution Guidelines](http://img.shields.io/badge/CONTRIBUTING-Guidelines-blue.svg)](./CONTRIBUTING.md)
 
-The Archives Unleashed Toolkit is an open-source platform for analyzing web archives built on [Apache Spark](http://spark.apache.org/), which provides powerful tools for analytics and data processing. This toolkit is part of the [Archives Unleashed Project](http://archivesunleashed.org/).
+The Archives Unleashed Toolkit is an open-source platform for analyzing web archives built on [Apache Spark](http://spark.apache.org/), which provides powerful tools for analytics and data processing. The Toolkit is part of the [Archives Unleashed Project](http://archivesunleashed.org/).
 
```

Comment:
This looks great! The only suggestion I have is I wonder if there's any way to signal even more strongly that users should visit our docs.

Like a standalone paragraph line:

```
Learn more about the toolkit and how to use it by visiting our [comprehensive documentation](https://aut.docs.archivesunleashed.org/).
```

I know it's elsewhere and there's the badge above, but this might help sell it.